packages/fxa-shared/db/models/auth/prune-tokens.ts

```diff
@@ -10,34 +10,17 @@ import { BaseAuthModel, Proc } from './base-auth';
  * Manages pruning of stale tokens.
  */
 export class PruneTokens extends BaseAuthModel {
-  /** Current state that the token pruner is in. */
-  public get state() {
-    return this._state;
-  }
-
-  /**
-   * Gets the timeout id for any pending prune operation.
-   */
-  public get currentTimeoutId() {
-    return this._timeoutId;
-  }
-
-  private _state: 'open' | 'pending' | 'active';
```

Comment:
We don't need to worry about de-bouncing calls anymore. Pruning now happens out of process in a controlled way.